includes/admin/upgrades/upgrade-functions.php

```diff
@@ -281,6 +282,32 @@ function edd_show_upgrade_notices() {
 					?>
 				</div>
 				<?php
+			} else {
+
+				// The orders database table is missing (we assume all primary tables have failed to create).
+				$message          = __( 'Easy Digital Downloads was unable to create the necessary database tables. Your site may not meet the minimum requirements for EDD 3.0.', 'easy-digital-downloads' );
+				$database_version = $wpdb->db_version();
+
+				// The database version is the problem.
+				if ( version_compare( $database_version, '5.6', '<' ) ) {
+					$message .= ' ' . sprintf(
+						/* translators: MySQL database version, do not translate */
+						__( 'Please contact your host and ask them to upgrade your MySQL version (your current version is %s).', 'easy-digital-downloads' ),
```

Comment:
We can't leave them without what to upgrade to. Telling them that they need to upgrade their version could lead them to contact our support asking what version. We should link off to our recommended wordpress hosting page, which lists our requirements.

Reply:
Sure, I can link to that...it makes me think that there is somewhere else that we want to link to something similar and it goes to the general EDD overview doc...which does have the minimum requirements, but all the way at the bottom. Updating that to use the same link would be better. I just have to remember where it was.